common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1529,7 +1529,7 @@ def reset_problem(self, _data):
             return {
                 'success': False,
                 # Translators: A student must "make an attempt" to solve the problem on the page before they can reset it.
-                'error': _("Refresh the page and make an attempt before resetting."),
+                'msg': _("You must make an attempt before resetting."),
```

Comment:
Good points - how about: "You must submit an answer before you can select Reset."